configs/default/RUSH-BLADE_F7_HD.config

```diff
@@ -0,0 +1,113 @@
+# version
+# Betaflight / RUSHFCHD (RF7H) 4.2.1 Aug 13 2020 / 00:11:54 (caa0d683c) MSP API: 1.43
```

Comment:
This will not work:
- the `# Betaflight...` line needs to be the first one in the file;
- this line needs to be for the appropriate 'STM32...' unified target, or else configurator will not know which target to flash.

Reply:
oh forgot to change this